### PR TITLE
Feature/create order model

### PR DESCRIPTION
**Pull request by DmitrS**

### CODE REVIEW

File: app/models/order.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Order < ApplicationRecord
+  belongs_to :client
+  belongs_to :driver
+  validates :from, presence: true
+  validates :to, presence: true
+  validates :state, presence: true
+  validates :price, presence: true, numericality: { greater_than: 0.01 }
```
Comment (EvgeniiK) [inline app/models/order.rb]: Where is 'end' ?

Comment (DmitrS) [inline app/models/order.rb]: Fixed. commit 96289c6661d2a87aae02fd3c9179e15b0ee54c98

**Comment by EvgeniiK** [comment]
Please squash commits into one by git.

File: app/models/order.rb (hunk 1 above)
Comment (EvgeniiK) [inline app/models/order.rb]: Where is 'end' again?)

Comment (DmitrS) [inline app/models/order.rb]: Done. I hope so at least. commit 78c5610e4148e572ce5925a72b895a228f5e0e4d

**Comment by DmitrS** [comment]
Fixed conflict in schema.rb

**Pull request merged**